tests/pipelines/main.nf.test

```diff
@@ -109,6 +117,94 @@ nextflow_pipeline {
         }
     }
 
+    test("Small-scale test of full pipeline with all samples same partition, tree comparison"){
+        tag "pipeline"
+        tag "pipeline_all"
```

Comment:
Is the double tag here intentional?

Reply:
Yes. I wanted to have a separate tag so I could run this test alone.